### PR TITLE
cleanup(angular): replace wrapAngularDevkitSchematics for component

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

We currently rely on @schematics/angular to generate our components 

## Expected Behavior

We should have everything we need to generate components ourselves without having to wrap @schematics/angular

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__ [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@Component({<% if(!skipSelector) {%>
+  selector: '<%= selector %>',<%}%><% if(standalone) {%>
+  standalone: true,
+  imports: [CommonModule],<%}%><% if(inlineTemplate) { %>
+  template: `
+    <p>
+      <%= fileName %> works!
+    </p>
+  `<% } else { %>
```
Comment (leosvelperez) [inline packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__:12]:
```suggestion
  template: `<p><%= fileName %> works!</p>`<% } else { %>
```

File: packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__ [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      }
+    `<% } %>
+  ]<% } else if (style !== 'none') { %>,
+  styleUrls: ['./<%= fileName %><%= type ? '.' + type : '' %>.<%= style %>']<% } %><% if(!!viewEncapsulation) { %>,
+  encapsulation: ViewEncapsulation.<%= viewEncapsulation %><% } if (changeDetection !== 'Default') { %>,
+  changeDetection: ChangeDetectionStrategy.<%= changeDetection %><% } %>
+})
+export class <%= className %><%= typeClassName %> {
+
+}
```
Comment (leosvelperez) [inline packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__:27]:
```suggestion
export class <%= className %><%= typeClassName %> {}
```

File: packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__ [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+@Component({<% if(!skipSelector) {%>
+  selector: '<%= selector %>',<%}%><% if(standalone) {%>
+  standalone: true,
+  imports: [CommonModule],<%}%><% if(inlineTemplate) { %>
+  template: `
+    <p>
+      <%= fileName %> works!
+    </p>
+  `<% } else { %>
+  templateUrl: './<%= fileName %><%= type ? '.' + type : '' %>.html'<% } if(inlineStyle) { %>,
```
Comment (leosvelperez) [inline packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__:13]: I guess if `style === 'none'` we don't want to generate inline styles either, isn't it?

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__ (hunk 3 above)
Comment (Coly010) [reply packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__:13]: no, i think you need one or the other. this just matches what the Angular CLI generates right now anyway too

File: packages/angular/src/generators/component/schema.json [hunk 4]
```diff
@@ -30,6 +30,20 @@
...
       "x-prompt": "What name would you like to use for the component?"
     },
+    "prefix": {
+      "type": "string",
+      "description": "The prefix to apply to the generated component selector.",
+      "alias": "p",
+      "oneOf": [
+        {
+          "maxLength": 0
+        },
```
[the remark this answers is not in the document]
Comment (Coly010) [reply packages/angular/src/generators/component/schema.json:40]: i think this is just a bad copy from the angular cli

**Review by leosvelperez — APPROVED** [review]

**Pull request merged**